packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Breaking Changes
+* Now requires users to enter separators themselves such as '/' or '-' depending on format.
```

Comment:
Why would this be considered a breaking change? I was thinking that this would be a Fix to not accept values that do not conform to the provided date format.

Reply:
Forgot to change the changelog after you clarified it. oops. Fixed now https://github.com/cerner/terra-framework/pull/851/commits/97ec7aef04215c4ca8bc967cb46af0d6250d6d04